package.json

```diff
@@ -4,7 +4,7 @@
   "version": "1.0.0",
   "scripts": {
     "start": "expo start",
-    "start-new": "EXPO_PUBLIC_CACHE_KEY=$(date +%s) expo start",
+    "start-new": "EXPO_PUBLIC_CACHE_KEY=$(date +%s) EXPO_PUBLIC_QUEUE_CACHE_KEY=$(date +%s) expo start",
```

Comment:
wanna account for:

- EXPO_PUBLIC_BATCH_CACHE_KEY
- EXPO_PUBLIC_ACTIVE_QUEUE_CACHE_KEY
- EXPO_PUBLIC_COMPLETED_QUEUE_CACHE_KEY

Reply:
Yes, it is accounted for -- the above is the old change. I've pushed a commit to include the one you've mentioned.